mixer/template/tracespan/tracespan.proto

```diff
@@ -38,7 +38,7 @@ syntax = "proto3";
 //     startTime: request.time
 //     endTime: response.time
 //     clientSpan: (context.reporter.kind | "inbound") == "outbound"
-//     rewriteClientSpanId: false
+//     rewriteClientSpanId: "false"
```

Comment:
The field is typed as `bool` in the proto. Why do we need quotes around it?

Reply:
Hi @kyessenov, sorry for my late response! That makes me confused as well:-)

Actually, I just followed the original sample snippet documented above in the comments, and got the error during runtime: `* instance='mytrace.instance.istio-system': json: cannot unmarshal bool into Go value of type string`.

So, apparently the value w/o quotes doesn't work. I think that's probably because of the transformation from json, while the Go type of the value is actually string... Haven't had chance to dig in why/where it comes out, but it looks most codes are auto-generated...

Any idea @douglas-reid, since you pointed out this issue for me [here](https://discuss.istio.io/t/apply-zipkin-adapter-w-tracespan-template-not-working/3071/2?u=morningspace) :-)

Reply:
it definitely has something to do with how the template generation and json transformation is working. but i haven't tried to track it down fully.